datajunction-query/config.djqs.yml

```diff
@@ -5,13 +5,22 @@ engines:
     uri: duckdb:////code/docker/default.duckdb
     extra_params:
       location: /code/docker/default.duckdb
+  # - name: trino
+  #   version: 451
+  #   type: sqlalchemy
+  #   uri: trino://domain:443/tpch/sf1
+  #   extra_params:
+  #     http_scheme: https
+  #     user: admin
+  #     password: 123
+  #     auth_type: dynamic
```

Comment:
These commented lines were added by accident I'm assuming.

Reply:
oh I added them intentionally to give an example

Reply:
I think it would be better to put that in the docs, maybe in the authentication section you can start a section on auth between the query service and query engines, starting with Trino. Also I see `auth_type` here but don't see where that's used in the code.